### ISSUE TITLE
Missing README.md file in the root dir of the project

### ISSUE THREAD

**Issue by kshithijiyer** [openshift/ci-tools#17]

**Comment by bbguimaraes** [comment]
Yeah, to keep the script simple, I just renamed the original files from each repository when I merged. A simple file with links to each of the old `README.md`s should be enough (we can merge `GENERATOR.md` and `CI_OPERATOR_PROWGEN.md`):

https://github.com/openshift/ci-tools/blob/master/CI_OPERATOR.md
https://github.com/openshift/ci-tools/blob/master/CI_OPERATOR_PROWGEN.md

**Comment by bbguimaraes** [comment]
Actually, never mind, I thought `GENERATOR.md` and `CI_OPERATOR_PROWGEN.md` had come from different repositories, but now I see they were intentionally separate from the start.

**Comment by petr-muller** [comment]
/lifecycle-remove stale
/assign

I'll write a quick readme today

**Comment by alvaroaleman** [comment]
repo now has a readme
/close
